src/constraints/angular_constraint.rs

```diff
@@ -67,9 +67,11 @@ pub trait AngularConstraint: XpbdConstraint<2> {
         // Apply rotational updates
         if body1.rb.is_dynamic() {
             *body1.rotation += Self::get_delta_rot(rot1, inv_inertia1, p);
+            *body1.rotation.0 = *body1.rotation.0.normalize();
```

Comment:
`rotation.0` is the quat itself, so it can't be dereferenced, which is making CI fail. Same in other places
```suggestion
            body1.rotation.0 = body1.rotation.normalize();
```

Reply:
Hmmm. I did that _for some reason_ that I don't remember now. I'll check and fix the PR. Thanks!